### PR TITLE
fix(demoNav): fix "Other Links" auto expand

### PR DESCRIPTION
**Pull request by CITguy**
Prevent "Other Links" from auto expanding.

### BEFORE
**Expanded on load**
screen shot 2015-08-20 at 2 30 00 pm

### AFTER
**Collapsed on load**
screen shot 2015-08-20 at 2 30 07 pm

- [x] DEV LGTM
- [x] Design LGTM

### CODE REVIEW

**Comment by parlarjb** [comment]
LGTM

**Comment by glynnis** [comment]
LGTM

File: demo/demo.js [hunk 1]
```diff
@@ -171,7 +171,6 @@ angular.module('demoApp', ['encore.ui', 'ngRoute'])
                 },
                 {
                     linkText: 'Other Links',
-                    href: '',
```
Comment (Droogans) [inline demo/demo.js]: I did this. Sorry.

**Comment by CITguy** [comment]
Closing. Upcoming PR will be removing nav section anyway

**Pull request closed without merging**